trimmerdecorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    name = @nameable.correct_name
+    name.length > 10 ? name[0, 9] : name
+  end
```

Comment:
- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#prepare-capitalizedecorator-and-trimmerdecorator) the output should have a maximum of 10 characters. However, your output is 9 characters maximum 😞. You may consider changing to `[0, 10]` to extract up to 10 characters if the length is greater than 10 ☺️.